docs/gen-ai/azure-ai-inference.md

```diff
@@ -21,6 +15,19 @@ The Semantic Conventions for [Azure AI Inference](https://learn.microsoft.com/az
 <!-- markdownlint-capture -->
 <!-- markdownlint-disable -->
 
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+Semantic Conventions for [Azure AI Inference](https://learn.microsoft.com/azure/ai-studio/reference/reference-model-inference-api) client spans extend and override the semantic conventions for [Gen AI Spans](gen-ai-spans.md).
+
+`gen_ai.system` MUST be set to `"az.ai.inference"` and SHOULD be provided **at span creation time**.
+
+**Span name** SHOULD be `{gen_ai.operation.name} {gen_ai.request.model}` when the
+model name is available and `{gen_ai.operation.name}` otherwise.
```

Comment:
was this defined somewhere previously? (just for whether to call out to gen ai approvers)

Reply:
unfortunately it wasn't, but seems necessary since `gen_ai.request.model` is not always available. 

@open-telemetry/semconv-genai-approvers could you please review this (and other GenAI changes)?

Reply:
resolving, it's fine to follow-up separately with genai folks, don't want to hold up the PR due to pending merge conflicts